README.md

```diff
@@ -12,11 +12,41 @@ A plugin for Terraform enabling it to manipulate
 
 ## Usage
 
+This terraform plugin supports basic connections to the ETCD endpoint,
+the HTTP API endpoint, and over SSH
+
+There is minimal configuration currently supported so the ETCD and API clients
+are attempting to connect directly without SSL
+
+The configuration value 'driver' defaults to 'tunnel' but can be configured with:
+ * etcd
+ * api
+
+EX:
+
+```
+provider "fleet" {
+  driver = "etcd"
+  endpoint = "http://192.168.0.1:4001"
+  // etcd_key_prefix can be ommited to use the default value
+  etcd_key_prefix = "/_some/_weird/etcd/prefix"
+  // connection_retries defaults to 12
+  connection_retries = 9000
+}
+```
+
+```
+provider "fleet" {
+  driver = "api"
+  endpoint = "http://192.168.0.1:4001"
```

Comment:
`driver = "api"` means to use the fleet HTTP API, not frobbing etcd directly, right? If so, it's pretty unlikely the endpoint would be on port 4001 in this case.